101-vm-simple-windows-visualstudio2019/azuredeploy.json

```diff
@@ -0,0 +1,259 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "type": "string",
+            "defaultValue": "[concat('vm', uniqueString(resourceGroup().name))]",
+            "metadata": {
+                "description": "The name of you Virtual Machine."
+            }
+        },
+        "VmSize": {
+            "type": "string",
+            "allowedValues": [
```

Comment:
why are we limiting to these VM sizes?

Reply:
@bmoore-msft limiting size to a VM we know will work as all A series VMs have been depreciated for Azure for Student so we are selecting a base machine which can be installed and scaled if required via the Portal

Reply:
I think this is going to be too limiting - we're excluding new sizes, "S" sizes and it won't account for quota, capacity, policy restrictions.  I think we need to leave this unbounded and can add appropriate documentation to the gettingstarted.md